app/views/facility_estimates/_estimate_detail.html.haml

```diff
@@ -5,7 +5,11 @@
     = hidden_field_tag "#{id_prefix}[duration_unit]", estimate_detail.duration_unit
   = hidden_field_tag "#{id_prefix}[_destroy]", false, class: 'destroy-field'
 
-  %td= estimate_detail.product.try(:name)
+  %td
+    - product_name = estimate_detail.product.try(:name)
+    - if estimate_detail.product.try(:facility) && estimate_detail.product.facility != current_facility
```

Comment:
can we always assume that `estimate_details.product` will be set? I would think so

Reply:
I think you're right